schema/workbook.json

```diff
@@ -192,10 +192,6 @@
                         "title": {
                             "$ref": "#/definitions/title",
                             "description": "title displayed above the grid/chart"
-                        },
-                        "exportParameters": {
-                            "type": "boolean",
-                            "description": "indicates if the parameters defined inside this group are visible outside the group"
                         }
```

Comment:
why did this get removed?

Reply:
I think this is in the wrong section, it's already in the groups definition. This was removed from the metrics definition